.github/workflows/publish-image.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Authenticate and set context
-        if: endsWith(github.ref, '/dev') && matrix.license == 'e4ca1d'
+        if: (endsWith(github.ref, '/dev') && matrix.license == 'e4ca1d' || endsWith(github.ref, '/main') && matrix.license == 'eb75ad')
```

Comment:
Are you certain this block handles the conditions properly? looks like the `||` might need to be `) || (` .